yearzero/YZCombat.js

```diff
@@ -3,9 +3,31 @@ const Util = require('../utils/Util');
 const YZInitiative = require('./YZInitiative');
 const Sebedius = require('../Sebedius');
 const { SUPPORTED_GAMES } = require('../utils/constants');
+const { __ } = require('../lang/locales');
 
-class YZCombat {
+/**
+ * @typedef CombatOptions
+ * @type {Object} options of the combat
+ * @property {boolean} dynamic Is the combat initiative dynamic? //TODO: correct description
+ * @property {string} lang The language code of the current combat instance
+ * @property {string} name The name for the this combat instance
+ * @property {boolean} turnnotif Turn notifications enabled status
+ */
 
+class YZCombat {
+	/**
+	 * A combat instance
+	 * @param {string} channelId The Discord TextChannel ID of the channel were the combat instance is ongoing.
+	 * @param {string} summaryMessageId The Discord Message ID of the summary message.
+	 * @param {string} dmId The GM's Discord User ID.
+	 * @param {CombatOptions} options The initial options of the combat
+	 * @param {Discord.Message} ctx The Discord Message
+	 * @param {YZCombatant[]} combatants Combatants
+	 * @param {?Iterable} initiatives Array of Key-Value pairs, K: {number} initiative value, V: {string} Combatant's ID.
+	 * @param {number} roundNum The current round.
+	 * @param {number} currentIndex The current initiative index (float value).
+	 * @param {string} game The game set for this combat instance.
```

Comment:
```suggestion
	 * A combat instance.
	 * @param {string} channelId The Discord TextChannel ID of the channel were the combat instance is ongoing
	 * @param {string} summaryMessageId The Discord Message ID of the summary message
	 * @param {string} dmId The GM's Discord User ID
	 * @param {CombatOptions} options The initial options of the combat
	 * @param {Discord.Message} ctx The Discord Message
	 * @param {YZCombatant[]} combatants Combatants
	 * @param {?Iterable} initiatives Array of Key-Value pairs, K: {number} initiative value, V: {string} Combatant's ID
	 * @param {number} roundNum The current round
	 * @param {number} currentIndex The current initiative index (float value)
	 * @param {string} game The game set for this combat instance
```

Reply:
Why remove the dots here? They are added everywhere else...

Reply:
I changed it to make it the same as the others int [this commit](https://github.com/Stefouch/sebedius-yearzero-discord-bot/pull/106/commits/02d2235bc4dbd46f6f5286f449825c531fc1422b).